packages/atomic/cypress/support/commands.ts

```diff
@@ -1,26 +1,39 @@
 // Must be declared global to be detected by typescript (allows import/export)
+
+import {AnalyticsTracker, SearchEventRequest} from '../utils/analyticsUtils';
+
 // eslint-disable @typescript/interface-name
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace Cypress {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     interface Chainable<Subject> {
-      getAnalyticsAt(selector: string, order: number): Chainable<unknown>;
       getTextOfAllElements(selector: string): Chainable<unknown>;
       // https://github.com/cypress-io/cypress-documentation/issues/108
       state(key: string): CypressRequest[];
       shouldBeCalled(urlPart: string, timesCalled: number): Chainable<unknown>;
+      expectSearchEvent(actionCause: string): Chainable<SearchEventRequest>;
```

Comment:
Alternatively, we could take a `Partial<SearchEventRequest>` as a parameter to eliminate the need for `then`. I decided to keep it simple here.